### ISSUE TITLE
Cleaner UI with dropdown details

### ISSUE THREAD

**Issue by Tris20** [Tris20/FairFareFinder#50]
I'm thinking similar to what ben had, that we do a dropdown thing, where this is the main view:
![image](https://github.com/Tris20/FairFareFinder/assets/6933318/c1bab8f8-6497-4b3c-b894-cfd36142d03a)

but if you click on one of the rows, it expands downwards to show flights, hotels, cost of living etc, events etc

One thing I would maybe add to the initial view is $$$ $$ or $ to represent overall comparative cost. i.e. if hotel expenseive and flights expesive and meal for two expensie $$$, if flights sometimes cheap, hotel cheap and meal for 2 cheap then $

so basically like this:
![image](https://github.com/Tris20/FairFareFinder/assets/6933318/8c95bb19-e625-48a9-aa2a-d66c1b25f64d)

and then click to expand and see details
